packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -200,3 +200,24 @@ export type Translation = {
 export const translationGuard: z.ZodType<Translation> = z.lazy(() =>
   z.record(z.string().or(translationGuard))
 );
+
+/**
+ * Hooks
+ */
+
+export type HookConfig = {
+  /** We don't need `type` since v1 only has web hook */
+  // type: 'web';
+  /** Method fixed to `POST` */
+  url: string;
+  /** Additional headers that attach to the request */
+  headers: Record<string, string>;
+  /** Only retry when status code >= 500 */
+  retries: number; //
+};
+
+export const hookConfigGuard: z.ZodType<HookConfig> = z.object({
+  url: z.string(),
```

Comment:
Do we need to guard the valid URL format

Reply:
I was thinking about it. But found the format doesn't matter since as long as the url is accessible it'll be fine (imaging an internal network with some custom format)